### ISSUE TITLE
Create debug url patterns for django's testing server

### ISSUE THREAD

**Issue by joshglenn** [sheppard/django-media-thumbnailer#1]
If the directives in `django-media-thumbnailer/dmt/conf/example.conf` could be written in the urls.py file such that the thumbnailer would work with django's testing server, that would be a plus for some.

**Comment by sheppard** [comment]
Yes, the dependency on Apache is one reason I haven't been sure this was ready for PyPI submission.  You can get around this for now by replacing e.g. `/media/160` with `/generate/160` in all of the templates, or adding this rule:

``` python
    url(r'^media/', include('dmt.urls')),
```

Either way you'll end up calling the thumbnail function every time (rather than benefiting from the filesystem "cache" the Apache config leverages), but at least it will work.
